.examples/docker-compose/with-mariadb/docker-compose.yaml

```diff
@@ -1,7 +1,7 @@
 version: '2.4'
 services:
   mariadb:
-    image: mariadb:10.5
+    image: mariadb:10.6
```

Comment:
sollen wir hier auf der 10.5 bleiben, da wir in der readme sagen das wir 10.5 unterstützen? Wirkt sonst ggf flaky, auch wenn es funktioniert.

Reply:
eventuell sollten wir dann überall auf die 10.6 gehen :-)

Reply:
oder man will bei 10.5 bleiben sehe ich gerade 10.6 ist noch nicht GA